### DISCUSSION TITLE
Does Salvo support session-based authentication?

### DISCUSSION THREAD

**Discussion by cameronelliott** [salvo-rs/salvo#843] [General]
Hey, I just discovered Salvo, and it looks cool.
I am trying to understand the authentication options, and I saw the basic, and JWT examples in the docs.

I generally lean towards session-based auth, over basic or JWT, unless I have a use case where JWT really stands out. (like multi-server backends)

Does Salvo support session-based auth?
Thanks

**Comment by chrislearn** [comment]
https://github.com/salvo-rs/salvo/blob/main/examples/session-login/src/main.rs
